modulefiles/module_base.orion

```diff
@@ -50,7 +50,7 @@ module load prod_util/1.2.0
 ##
 module use /apps/contrib/NCEPLIBS/lib/modulefiles
 module load netcdfp/4.7.4.release
-module load esmflocal/8.1.0.21bs.release
+module load esmflocal/8.1.0.27bs.release
```

Comment:
can we also update this for module_base.hera?

Reply:
Got it! I will.